### DISCUSSION TITLE
Successor or Raptor plugin?

### DISCUSSION THREAD

**Discussion by buremba** [trinodb/trino#13956] [Q&A]
I realized that the `presto-raptor` plugin is renamed to `presto-raptor-legacy`. If I'm not mistaken, there was an ongoing rewrite of the Raptor plugin in the previous repository which was going to be released as a new module a while ago. 

What's the status of that module, if there is no development, is there any Presto plugin that we can count on as an alternative to Raptor plugin?

**Comment by findepi** [comment]
@buremba please see discussion on slack:

https://prestosql.slack.com/archives/CFLB9AMBN/p1569521182270900?thread_ts=1569432769.206500&cid=CFLB9AMBN

https://prestosql.slack.com/archives/CFLB9AMBN/p1573040231471800

**Comment by buremba** [comment]
Thanks for the links @findepi! I wrote to the relevant thread on Slack but let me write here also:

We recently started discussing implementing the Raptor connector again for the new AWS product Athena Query Federation. Is anyone willing to collaborate?
